node/typescript.ts

```diff
@@ -0,0 +1,24 @@
+import lambda = require('aws-lambda');
```

Comment:
Why use `require()` here?

Reply:
Typescript imports and exports work slightly differently as up until a latest version you could only export one big namespace with a set of members or one default export, but not both. 

From the docs: `


From the TS docs: "When exporting a module using export =, TypeScript-specific import module = require("module") must be used to import the module."